### PR TITLE
Fix flaky kill_before_submit test for lsf

### PR DESCRIPTION
**Pull request by eivindjahren**
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: tests/ert/unit_tests/scheduler/test_lsf_driver.py [hunk 1]
```diff
@@ -1253,16 +1236,39 @@ async def test_that_kill_before_submit_is_finished_works(
...
+        ),
+        encoding="utf-8",
+    )
+    job_path.chmod(job_path.stat().st_mode | stat.S_IEXEC)
+
     # Allow submit and kill to be interleaved by asyncio by issuing
     # submit() in its own asyncio Task:
     asyncio.create_task(
         driver.submit(
             # The sleep is the time window in which we can kill the job before
```
Comment (berland) [inline tests/ert/unit_tests/scheduler/test_lsf_driver.py:1268]: comment on sleep is now irrelevant.

**Comment by berland** [comment]
Has this been tested "live" on real compute clusters?

**Comment by eivindjahren** [comment]
> Has this been tested "live" on real compute clusters?

Not yet :D. I'll try it.

**Comment by eivindjahren** [comment]
@berland Just checked and it worked to run this on tgx.

**Review by berland — APPROVED** [review]
Lets roll! 🛼

**Comment by eivindjahren** [comment]
@berland Seems like there is actually something wrong with the slurm driver: 

https://github.com/equinor/ert/actions/runs/11105990094/job/30853371751?pr=8807#step:8:141

The logs seem to indicate that the realization is marked as successful even though it is killed.

**Pull request merged**